packages/hs-react-ui/src/components/Card/Card.tsx

```diff
@@ -144,6 +144,10 @@ const Card = ({
   disableFeedback,
   feedbackType = FeedbackTypes.ripple,
 }: CardProps): JSX.Element | null => {
+  const hasContent = Boolean(children);
+  const hasFooter = Boolean(footer);
+  // eslint-disable-next-line no-param-reassign
+  headerProps = { ...headerProps, hasContent, hasFooter };
```

Comment:
I'd just declare hasContent and hasFooter above the spread, and include them on the component declaration (Line 194) - and also pass both of those variables to the footer/body as well for people to utilize when writing their own styled components

Reply:
On it!

Reply:
@aVileBroker And should I go ahead and make a hasHeader to pass the Body and Footer?

Reply:
Yeah I think that's a good idea